leaderboard/models/user.py

```diff
@@ -1,13 +1,14 @@
-from sqlalchemy import Column, BigInteger, Integer, String, select, func, and_
-from sqlalchemy.orm import column_property
+from sqlalchemy import Column, BigInteger, Integer, String
 from db import get_db
 
+
 class User(get_db().Base):
     __tablename__ = 'user'
     id = Column(Integer, primary_key=True)
     # nickname should be unique?
     nickname = Column(String, nullable=False)
     email = Column(String)
+    last_update = Column(BigInteger, default=0)
```

Comment:
I should really get rid of this field.  There's no way to use it from the server side as the client only passes in an OAuth2 bearer token.